src/utils/index.ts

```diff
@@ -120,7 +120,10 @@ export function shouldRenderEditButton(editButton: IConfigEditButton): boolean {
         const currentLocation = new URL(window.location.href);
         const cslpButtonQueryValue =
             currentLocation.searchParams.get("cslp-buttons");
-        if (cslpButtonQueryValue !== null)
+        if (
+            cslpButtonQueryValue !== null &&
+            editButton.includeByQueryParameter !== false
```

Comment:
```suggestion
            editButton.includeByQueryParameter
```

Reply:
i've added this type of condition knowingly, initially if customer not added this key into edit button object, `editButton.includeByQueryParameter` comes as undefined which we are handling later on by assigning the default value to it but the time when we interpret this condition the value will undefined, that's why i added the condition for "false" value only